### PR TITLE
Adding `logger` option for `express-fileupload`

### PR DESCRIPTION
**Pull request by RobinTail**
I'm adding the type of a new option I [added myself](https://github.com/richardgirges/express-fileupload/pull/371) to that package. The changes are released in version [1.5.0](https://github.com/richardgirges/express-fileupload/releases/tag/v1.5.0). Tests are updated, and I tested the changes manually in a real project.

Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:

- [X] Provide a URL to documentation or source code which provides context for the suggested changes: [PR 371: Custom logger](https://github.com/richardgirges/express-fileupload/pull/371)
- [X] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the `package.json`.

### CODE REVIEW

File: types/express-fileupload/index.d.ts [hunk 1]
```diff
@@ -135,6 +135,11 @@ declare namespace fileUpload {
          * @default false
          */
         debug?: boolean | undefined;
+        /**
+         * Customizable logger to write debug messages to.
+         * @default console
+         */
+        logger?: { log: (msg: string) => void } | undefined;
```
Comment (peterblazejewicz) [inline types/express-fileupload/index.d.ts:142]: you mean something like (taking shape to match contract):
```ts
logger?: { log: Console["log"] } | undefined;
```

Comment (RobinTail) [reply types/express-fileupload/index.d.ts:142]: Ok. Can do it this way.

Comment (RobinTail) [reply types/express-fileupload/index.d.ts:142]:
```suggestion
        logger?: { log: Console["log"] } | undefined;
```

**Review by peterblazejewicz — APPROVED** [review]
@RobinTail thanks!

**Comment by RobinTail** [comment]
Ready to merge , @typescript-bot

**Pull request merged**